src/main/java/bio/terra/common/iam/AuthHeaderKeys.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.common.iam;
+
+/** Keys for auth headers */
+public enum AuthHeaderKeys {
+  OIDC_ACCESS_TOKEN("OIDC_ACCESS_token"),
```

Comment:
Could these just be static final strings? It seems they are only used as a way to get strings.

Even if you leave it an enum, could it be moved inside of `ProxiedAuthenticatedUserRequestFactory`? Looks like that is the only place they are used.

Reply:
Done...  

This was a copy/paste, so I wasn't sure what value the Enum was adding over a String constant.  I also assume there was a thought that this might be reusable, but I also don't see anyone else using it...  We can always move it out in the future if there's a need to broaden the scope.

On a related note, I did want to move [LocalAuthenticatedUserRequestFactory](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/iam/LocalAuthenticatedUserRequestFactory.java) over from TDR, as well but balked due to the dependence on `ApplicationConfiguration`.  I would like to at least unify the parsing of the bearer token to use the regex logic I've added here for consistency.  What would be the idiomatic way to make this helper logic usable?

Reply:
The `Local...` version is only for testing. In TDR, the email and userid do not matter. I am not sure that is always the case. 

The idea is that in some test environment you are not running the front-end proxy that fills those fields in. So to have everything else work, you need a way to reliably fill in those fields.

Reply:
Without looking at the details, it might make sense to add the `LocalAuthenticatedUserRequestFactory` to a `testFixtures` part of the TCL. It'd be nice to share testing code as well, particularly as it's relevant to the common library. But we can certainly do that in a later PR.

Reply:
Yeah, decoupling from the local app config will take a little extra work, I'll consider revisiting when I get to the consuming the changes in TDR.